config/initializers/assets.rb

```diff
@@ -34,7 +34,8 @@
 )
 
 Rails.application.config.assets.paths += %W(
-  #{__dir__}/../../node_modules/govuk-frontend/assets/images
-  #{__dir__}/../../node_modules/govuk-frontend/assets/fonts
+  #{__dir__}/../../node_modules/govuk-frontend/govuk/assets/images
+  #{__dir__}/../../node_modules/govuk-frontend/govuk/assets/fonts
+  #{__dir__}/../../node_modules/govuk-frontend/
```

Comment:
Question: if the whole of the `node_modules/govuk-frontend` dir is a path, why do we need the images and fonts subdirs explicitly specified?

Reply:
I suspect this is because they're referenced without the full path (e.g. `my-font.otf` instead of `/govuk/assets/fonts/my-font.otf`)

Reply:
I've tested this and those extra paths are definitely needed. Weird. Oh well.